blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineImpl.java

```diff
@@ -164,4 +177,28 @@ protected static String getRecursivePathFromFullName(BluePipeline pipeline){
         return pipelinePath.toString();
     }
 
+    @Extension(ordinal = 1)
+    public static class PipelineFactoryImpl extends BluePipelineFactory {
```

Comment:
name could be maybe less redundant? DefaultPipelineFactory? Or just PipelineFactory?